2018/python/test_file_generator.py

```diff
@@ -2866,11 +2874,16 @@ def q604(self):
 		path = self.quality_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.state = lar.state.map(lambda x: random.choice(list(self.crosswalk_data.stateCode)))
+		lar.state = lar.state.map(lambda x: self.state_abbrev[random.choice(list(self.crosswalk_data.stateCode))])
+		print(lar.state)
 		#this implemenation sets all state codes to the same code and uses that to make a county list 
 		for index, row in lar.iterrows():
-			row["state"] = random.choice(list(self.crosswalk_data.stateCode))
-			row["county"] = random.choice(list(self.crosswalk_data.countyFips[self.crosswalk_data.stateCode!=row["state"]]))
+			state_code = random.choice(list(self.crosswalk_data.stateCode))
+			state_abbrev = self.state_abbrev[state_code]
+			row["state"] = state_abbrev
+			row["county"] = random.choice(list(self.crosswalk_data.countyFips[self.crosswalk_data.stateCode!=state_code]))
+			#Conforming the census tract to the new county code to pass v625 and v627. 
```

Comment:
Please change to: "forces the census tract to conform to an appropriate subset of county codes in order to pass v625 and v627.

Reply:
Sure! This change will be reflected in the next commit.